requirements-test.py2.txt

```diff
@@ -0,0 +1,7 @@
+django-mptt~=0.9.0
+mock~=3.0.0
+pytest-django~=3.10.0
+pytest-pythonpath~=0.7.0
+pytest~=4.6.11
+pytz==2020.1
+six~=1.15.0
```

Comment:
Since all but one dep versions are the same between 2 and 3 you can use `-r requirements-test.py3.txt` to include the contents of the other file in this file.